src/BinaryGapOrzech.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Created by Łukasz on 2017-04-09.
+ */
+public class BinaryGapOrzech {
+    public int solutionOrzech(int N){
+        String nat=Integer.toBinaryString(N);
+        List<String> myList = new ArrayList<String>(Arrays.asList(nat.split("")));
+        int binaryGap=0;
+        int biggestGap=0;
+
+        for(int i=0;i<myList.size() && myList.size()!=0;i++){
```

Comment:
Co to jest:D  `myList.size()!=0` przeciez jak lista bedzie miala rozmiar zero to 0<myList.size zwroci false i nie wejdzie do petli wywal to